swan-lake/learn-the-platform/run-in-the-cloud/function-as-a-service/azure-functions.md

```diff
@@ -76,60 +76,46 @@ The Azure Functions functionality is implemented as a compiler extension. Thus,
 ```
 $ bal build
 Compiling source
-	wso2/azure_functions_deployment:0.1.0
+        wso2/azure_functions_deployment:0.1.0
 
-Generating executables
-	@azure.functions:Function: hello
+Generating executable
+        @azure_functions:Function: get-hello
 
-	Run the following command to deploy Ballerina Azure Functions:
-	az functionapp deployment source config-zip -g <resource_group> -n <function_app_name> --src <package_dir>/target/bin/azure-functions.zip
-```
+        Execute the below command to deploy the function locally:
+        func start --script-root target/azure_functions --java
```

Comment:
```suggestion
        func start --script-root target/azure_functions --java
```
Shall we put the command inside a `bash` section?

Reply:
This is the output from the CLI. So I don't think we need to